pypika/queries.py

```diff
@@ -624,6 +625,16 @@ def replace(self, *terms):
             self._validate_terms_and_append(*terms)
         self._replace = True
 
+    @builder
+    def force_index(self, *terms):
+        for term in terms:
+            if isinstance(term, Field):
+                self._force_indexes.append(term)
+            elif isinstance(term, str):
+                self._force_indexes.append(Field(term, table=self._from[0]))
+        if len(self._force_indexes) == 0:
```

Comment:
this if statement is stateful, which might lead to undesirable results, if `force_index` is called twice on a query, for example. Maybe you want to validate it so that `force_index` can only be called once and requires at least one argument? You could also syntactically force this by accepting args like `def force_index(self, term, *terms)`, that way python would require always at least one argument